src/components/GameCards/GameCard.js

```diff
@@ -35,6 +40,47 @@ export default {
         && this.progresses[this.platform.code][this.game.id];
     },
 
+    releaseDate() {
+      const releaseDate = this.game
+        && this.game.release_dates
+        && this.game.release_dates.filter(
+          ({ platform }) => this.platform.id === platform,
+        )
+        && this.game.release_dates.filter(
+          ({ platform }) => this.platform.id === platform,
+        )[0].date;
```

Comment:
you can use `find` instead of `filter` to get the first instance, then you don't have to get it using `[0]`